worklogger.user.js

```diff
@@ -170,6 +170,10 @@ class P4U {
             element.accessKey = key;
         }
     }
+
+    static artefactField(){
```

Comment:
Máš ověřené, že je to ID stejné ve všech teritoriích a při přidání i editaci výkazu? V místě použití se nedělá null-check.

Reply:
Stejné ve všech teritoriích i při editaci.

Reply:
` if (!P4U.artefactField().value) { ` Tohle je null check ne?